### PR TITLE
Say command

### PR DESCRIPTION
**Pull request by noni3108**
The command would be q!say {#channel} {message}.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,569 +1,596 @@
...
+            color: 11253955,
+            description: `My commands are \`${config.prefix}help\`, \`${config.prefix}setrank <user> <rank number>\`, \`${config.prefix}promote <user>\`, \`${config.prefix}demote <user>\`, \`${config.prefix}fire <user>\`, \`${config.prefix}shout <msg>\`, and \`${config.prefix}clearshout\`.`,
+            author: {
+                name: message.author.tag,
+                icon_url: message.author.displayAvatarURL
+            }
+        }});
+    };
+  
+  if (command === "say") {
```
Comment (noni3108) [inline index.js]: Here is the command

**Comment by yogurtsyum** [comment]
Can you please give me a example of when this would be used?

**Comment by TypicallyShad** [comment]
Not going to lie, in my opinion, it's a really good command but isn't really required in qbot, I mean, unless this was a public bot for moderation or something, I would understand.

**Comment by yogurtsyum** [comment]
> Not going to lie, in my opinion, it's a really good command but isn't really required in qbot, I mean, unless this was a public bot for moderation or something, I would understand.

Shad is right on this one.

**Pull request closed without merging**